core/gesture.js

```diff
@@ -41,8 +41,9 @@ goog.require('goog.math.Coordinate');
 /**
  * NB: In this file "start" refers to touchstart, mousedown, and pointerstart
```

Comment:
["Nota bene"](https://en.wikipedia.org/wiki/Nota_bene), and I could have just used "Note".

No it should not be a doc comment.